Form/View/ExportViewType.php

```diff
@@ -117,9 +138,32 @@ public function getBlockPrefix()
      */
     public function generateResponse(View $view, Request $request)
     {
-        $response = new Response();
         $parameters = $this->getParameters($view, $this->formFactory, $request);
-        
+
+        if (isset($view->getOptions()['export_type']) or $view->getOptions()['export_type'] === 'dompdf') {
+            // instantiate and use the dompdf class
+            $dompdf = new Dompdf();
+            $dompdf->loadHtml($parameters['render']);
+
+            // (Optional) Setup the paper size and orientation
+            $dompdf->setPaper(
+                (isset($view->getOptions()['pdf_size']) and $view->getOptions()['pdf_size']) ? $view->getOptions()['pdf_size'] : 'A4',
+                (isset($view->getOptions()['pdf_orientation']) and $view->getOptions()['pdf_orientation']) ? $view->getOptions()['pdf_orientation'] : 'portrait'
+            );
+
+            // Render the HTML as PDF
+            $dompdf->render();
+
+            // Output the generated PDF to Browser
+            $dompdf->stream($parameters['filename'] ?? "document.pdf", [
+                'compress' => 1,
+                'Attachment' => ( isset($view->getOptions()['disposition'])  && $view->getOptions()['disposition']  === 'attachment')?1:0,
```

Comment:
there is a "space" too much before the && charachtesr, and before the === charachters. And seeing the && here gives more reasen to follow this convention in the other if checks.

Reply:
tha'is an expensive space ;-)